docs/learn/universal-profile/key-manager/execute-relay-transactions.md

```diff
@@ -356,65 +398,43 @@ const eip191Signer = new EIP191Signer();
 const { signature } = await eip191Signer.signDataWithIntendedValidator(
   keyManagerAddress,
   encodedMessage,
-  controllerPrivateKey,
+  '0xxxxxxxxx...xxxxxxxx', // controller / signer private key (65 bytes long)
 );
 ```
 
   </TabItem>
 
 </Tabs>
 
-After the signature has been generated, it can be sent to the third party to execute the transaction using [`executeRelayCall`](../../../contracts/contracts/LSP6KeyManager/LSP6KeyManager.md#executerelaycall) on the Key Manager of the profile. To verify all transaction parts, the third-party will need the following parameters:
+After the signature has been generated, it can be sent to the third party to be executed on the network. We will cover this in the last section [**"Step 3 - Execute the Relay Call"**](#step-4---execute-the-relay-call).
 
-- `signature` of the transaction payload
-- `abiPayload` including the operations
-- `nonce` of the signing controller
-- `validityTimestamps` of the transaction
-- `keyManagerAddress` of the signing profile
-
-## Execute the Relay Call
+## Step 3 - Execute the Relay Call
 
 :::info
 
-The following section shows how a third party can execute a transaction on behalf of another user.
+This logic must be implemented on the _relayer-side_ and the transaction sent to the network to be executed.
```

Comment:
Maybe
`This logic must be implemented on the _relayer-side_ where the transaction will be sent to the network to be executed.`